scripts/fence_setup.sh

```diff
@@ -13,4 +13,4 @@ update-ca-certificates
 
 fence-create sync --yaml user.yaml --arborist http://arborist-service
 
-rm -f /var/run/apache2/apache2.pid && /usr/sbin/apache2ctl -D FOREGROUND
+rm -f /var/run/apache2/apache2.pid && /dockerrun.sh
```

Comment:
I think you may also need to change this to remove the nginx.pid? `rm -f /var/run/nginx.pid` unsure if that's absolutely necessary though...